viser/_viser.py

```diff
@@ -218,7 +218,7 @@ class ViserServer(MessageApi):
     """Handle for manipulating the world frame axes (/WorldAxes), which is instantiated
     and then hidden by default."""
 
-    def __init__(self, host: str = "127.0.0.1", port: int = 8080):
+    def __init__(self, host: str = "0.0.0.0", port: int = 8080):
```

Comment:
also, is this an intentional change?

Reply:
This is intentional, 127.0.0.1 only allows your local machine to access it. 0.0.0.0 has the same effect on your machine, but allows you to access it on your local network by referencing your machine's IP.

Reply:
ok! I'm fine with the change, although we might want to revisit if we care about security practices